CRYOSMS/iocBoot/iocCRYOSMS-IOC-01/st-common.cmd

```diff
@@ -0,0 +1,24 @@
+##ISIS## Run IOC initialisation 
+< $(IOCSTARTUP)/init.cmd
+
+drvAsynIPPortConfigure("$(PORT=STREAM_PORT)", "localhost:$(EMULATOR_PORT=57677)")
+CRYOSMSConfigure("ASYN_PORT", "$(MYPVPREFIX)$(IOCNAME):")
+
+epicsEnvSet "STREAM_PROTOCOL_PATH" "$(CRYOSMS)/data"
+
+##ISIS## Load common DB records 
+< $(IOCSTARTUP)/dbload.cmd
+
+############################
+## Load our record instances
+############################
+dbLoadRecords("$(CRYOSMS)/db/cryosms.db","P=$(MYPVPREFIX)$(IOCNAME):,PORT=$(PORT=STREAM_PORT),ASYN_PORT=ASYN_PORT,RECSIM=$(RECSIM=0),DISABLE=$(DISABLE=0),PROTO=$(PROTO=cryosms.proto)")
```

Comment:
Why do we need to pass `PROTO` as an argument here, it does not look like it is used by the db?